content/docs/operating/security.md

```diff
@@ -163,6 +193,9 @@ environment variable as used by EC2 service discovery) may end up exposed due to
 code outside of our control or due to functionality that happens to expose
 wherever it is stored.
 
+Special care must be taken to prevent unauthorized reads or writes to
+configuration files that contain secrets.
```

Comment:
That's not really Prometheus specific, nor secret specific. I'd add something two paragraphs up that protecting the files on disk is the user's responsibility.

Reply:
ack